openwisp_monitoring/device/tests/test_admin.py

```diff
@@ -26,9 +26,12 @@ def test_device_admin(self):
         r = self.client.get(url)
         self.assertContains(r, '<h2>Status</h2>')
         self.assertContains(r, '<h2>Charts</h2>')
+        self.assertContains(r, '<h2>\n        Alert Settings\n    </h2>')
```

Comment:
I am not sure, I guess we can simply test whether the response contains `AlertSettings` (might not be a full proof approach since if in future more tabs are added which also contain the term `AlertSettings` then this might be obsolete.)